src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportControlLoader.java

```diff
@@ -66,12 +70,25 @@
     private static final String DTD_RESOURCE_NAME_1_2 =
         "com/puppycrawl/tools/checkstyle/checks/imports/import_control_1_2.dtd";
 
+    /** The resource for the configuration dtd. */
+    private static final String DTD_RESOURCE_NAME_1_3 =
+            "com/puppycrawl/tools/checkstyle/checks/imports/import_control_1_3.dtd";
+
     /** The map to lookup the resource name by the id. */
     private static final Map<String, String> DTD_RESOURCE_BY_ID = new HashMap<>();
 
     /** Name for attribute 'pkg'. */
     private static final String PKG_ATTRIBUTE_NAME = "pkg";
 
+    /** Name for attribute 'strategyOnMismatch'. */
+    private static final String STRATEGY_ON_MISMATCH_ATTRIBUTE_NAME = "strategyOnMismatch";
+
+    /** Value "allowed" for attribute 'strategyOnMismatch'. */
+    private static final String STRATEGY_ON_MISMATCH_ALLOWED_VALUE = "allowed";
+
+    /** Value "disallowed" for attribute 'strategyOnMismatch'. */
+    private static final String STRATEGY_ON_MISMATCH_DISALLOWED_VALUE = "disallowed";
+
```

Comment:
there should be constant for "delegateToParent" , default value is still valid value.